prototyping/wlutz/00-wlutz-with-pylinac.ipynb

```diff
@@ -336,7 +336,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.2"
+   "version": "3.7.9"
```

Comment:
is this intentional?
If so, what was the motivation?

Reply:
Just the Python installed on the machine that ran the notebook.